TerrorConsole/Assets/_Main/Scripts/Audio/AudioManager.cs

```diff
@@ -0,0 +1,61 @@
+using UnityEngine;
+using UnityEngine.Audio;
+using UnityEngine.Windows;
+
+namespace TerrorConsole
+{
+    public class AudioManager : Singleton<IAudioSource>, IAudioSource
+    {
+        [SerializeField] private AudioSource _sfxAudioSource;
+        [SerializeField] private AudioDatabase _audioDatabase;
+        [SerializeField] private AudioMixer _sfxMixer;
+
+        public float SFXVolume { get; private set; }
+
+        private void Start()
+        {
+            InitializeSFX();
+        }
+        private void InitializeSFX()
+        {
+            SFXVolume = PlayerPrefs.GetFloat("sfxVol", SFXVolume);
+            _sfxMixer.SetFloat("sfxVol", SFXVolume);
+        }
+        public void PlayDoorCloseSFX()
+        {
+            AudioData audioData = _audioDatabase.GetAudio("DoorCloseSFX");
+            _sfxAudioSource.PlayOneShot(audioData.audioClip, audioData.volume);
+        }
+
+        public void PlayDoorOpenSFX()
```

Comment:
Create at the bottom a helper method, this will allow us to make nice changes in the future without changing much code, after refactor the methods to use that one.

private void PlayOneShotSFX(string audioName)
      {
          AudioData audioData = _audioDatabase.GetAudio(audioName);
          _sfxAudioSource.PlayOneShot(audioData.audioClip, audioData.volume);
      }